src/components/Data/Leftnav/IndexBranch.vue

```diff
@@ -197,10 +195,17 @@ export default {
 </script>
 
 <style scoped lang="scss">
+.IndexBranch {
```

Comment:
Does this class name follow this project naming convention? and is it necessary for this component?

Reply:
The name of the class already existed but was not used. So I think he's respecting the naming convention. There's a more appropriate name?

Reply:
This name sucks. My fault. Fustigate Luca from 2016, please.

Reply:
If this class is not used, I believe it shouldn't be included in this PR (https://deviq.com/boy-scout-rule/).

Reply:
She wasn't. But she is now. Any suggestions for the name?